src/app/nada-consta/nada-consta.service.ts

```
@@ -35,15 +35,16 @@ export interface NadaConsta {
 
 @Injectable({ providedIn: 'root' })
 export class NadaConstaService extends CrudService<NadaConsta, number> {
-  constructor() {
-    const http = inject(HttpClient);
+  protected http: HttpClient;
+  constructor(http: HttpClient) {
```

ESLint

[error] Prefer using the inject() function over constructor parameter injection. Use Angular's migration schematic to automatically refactor: ng generate @angular/core:inject

(@angular-eslint/prefer-inject)